lib/dk_buttons_and_leds/dk_buttons_and_leds.c

```diff
@@ -54,7 +68,8 @@ static enum state state;
 static struct k_delayed_work buttons_scan;
 static button_handler_t button_handler_cb;
 static atomic_t my_buttons;
-static struct device *gpio_dev;
+static struct device *button_devs[ARRAY_SIZE(led_pins)];
```

Comment:
Should this be `ARRAY_SIZE(button_pins)` ?

Reply:
yes, good catch, thanks!

Reply:
fixed, not sure why mcuboot tests are failing

Reply:
These failures seem unrelated, they appear on other PRs too.